pages/client-libraries/rust.mdx

```diff
@@ -0,0 +1,714 @@
+
+                }
+                _ => continue
+            }
+        }
+    }
+}
+if let Err(e) = connection.commit() {
+    println!("Error: {}", e);
+}
+```
+
+The output of the code above is:
+```text
+Edge: [:LOVES {'id': 99}]
+Edge id: 455
+Edge start_node_id: 237
+Edge end_node_id: 238
+Edge type: LOVES
+Edge properties: {"id": Int(99)}
+Edge properties: Some(Int(99))
+```
+
+You can access the `Relationship` properties in the same way as you access the [Node](#process-the-node-result) properties. The only difference is that the `Relationship` has `start_id` and `end_id` properties, which represent the start and end node of the relationship.
+
+
+#### Process the Path results
+
+You can receive path from the database, using the following construct:
+
+```rust
+let _read_path = "MATCH p=(d:Developer)-[r:LOVES]->(t:Technology) RETURN p";
+let _columns = connection.execute(_read_path, None);
+while let Ok(result) =  connection.fetchall() {
+    for record in result {
+        for value in record.values {
+            match value {
+                Value::Path(path) =>
+                {
+                    println!("Path nodes: {:?}", path.nodes);
+                    println!("Path relationships: {:?}", path.relationships);
+                }
+                _ => continue
+            }
+        }
+    }
+}
+if let Err(e) = connection.commit() {
+    println!("Error: {}", e);
+}
+```
+
+Path will contain [Nodes](#process-the-node-result) and [Relationships[#process-the-relationship-result], that can be accessed in the same way as in the previous examples, by casting them to the relevant type.
+
+#### Types mapping and casting
+
+ Here is the full table of the mapping between Memgraph Cypher types and the types used in the Rust driver:
+
+|  Cypher Type | Driver Type        |
+|-------------:|:-------------------|
+|          Null| Null               |
+|       String | String             |
+|      Boolean | bool               |
+|      Integer | i64                |
+|        Float | f64                |
+|         List | Vec< Value >       |
+|          Map | HashMap< String, Value >|
+|         Node | Node               |
+| Relationship | Relationship       |
+|         Path | Path               |
+| UnboundRelationship | UnboundRelationship |
+|     Duration | Duration           |
+|         Date | NaiveDate          |
+|    LocalTime | NaiveTime          |
+|LocalDateTime | NaiveDateTime      |
+
+Keep in mind that Memgraph does not support timezones at the moment. 
+
+### Transaction management
+
+Transaction is a unit of work that is executed on the database, it could be some basic read, write or complex set of steps in form of series of queries. There can be multiple ways to mange transaction, but usually, they are managed automatically by the driver or manually by the explicit code steps. 
+Transaction management defines how to handle the transaction, when to commit, rollback, or terminate it.
```

Comment:
```suggestion
Transaction is a unit of work that is executed on the database, it could be some basic read, write or complex set of steps in form of series of queries. There can be multiple ways to mange transaction, but usually, they are managed automatically by the driver or manually by the explicit code steps. Transaction management defines how to handle the transaction, when to commit, rollback, or terminate it.
```